cmd/ufs/ufs.go

```diff
@@ -16,11 +16,17 @@ import (
 var (
 	ntype = flag.String("ntype", "tcp4", "Default network type")
 	naddr = flag.String("addr", ":5640", "Network address")
+	haddr = flag.String("http", ":8080", "Network address")
+	root  = flag.String("root", "/", "Set the root for all attaches")
```

Comment:
Do we want a root for http and a root for attaches?

Reply:
I was thinking of using the [attach root]( https://github.com/Harvey-OS/ninep/blob/8010cab8eac948c00149d228e8043bb950d124d6/filesystem/filesystem.go#L45)  as the http root. Should I just parse that flag once and pass it in to newUFS as an arg on line :35?

Reply:
it's confusing as worded for me. Attaches sounds like 9p.

"set the root for http service"
?